runtime/common/src/lib.rs

```diff
@@ -193,6 +193,23 @@ pub mod parachains {
 			pub use super::super::polkadot::asset_hub::*;
 		}
 	}
+
+	pub mod moonbase_alpha_relay {
+		pub mod moonbase_alpha {
+			use xcm::latest::{
+				Junction::{PalletInstance, Parachain},
+				Junctions::X2,
+			};
+	
+			pub const PARA_ID: u32 = 1000;
+			pub const BALANCES_PALLET_INDEX: u8 = 3;
+	
+			parachain_asset_location!(
+				DEV,
+				X2(Parachain(PARA_ID), PalletInstance(BALANCES_PALLET_INDEX),)
```

Comment:
Do we need the comma? 🤔 

```suggestion
				X2(Parachain(PARA_ID), PalletInstance(BALANCES_PALLET_INDEX))
```

Reply:
No, we can delete it 👍🏼